### PR TITLE
7.6.0 Support

### PR DESCRIPTION
**Pull request by aaerofeev**
Add support for elasticsearch 7.6.*
If you want, please create branch 7.6

### CODE REVIEW

**Comment by aaerofeev** [comment]
Build pass in travis

File: src/main/java/org/elasticsearch/index/analysis/keyboard/KeyboardAnalyzerProvider.java [hunk 1]
```diff
@@ -24,15 +24,18 @@
...
         super(indexSettings, name, settings);
+
+        analyzer = new WhitespaceAnalyzer();
+        analyzer.setVersion(version);
     }
 
     @Override
     public WhitespaceAnalyzer get() {
-        return INSTANCE;
+        return this.analyzer;
```
Comment (npapakha) [inline src/main/java/org/elasticsearch/index/analysis/keyboard/KeyboardAnalyzerProvider.java:39]: @aaerofeev Hi Could you please explain this change? Have you faced any issues with this analyzer at runtime?

Comment (aaerofeev) [reply src/main/java/org/elasticsearch/index/analysis/keyboard/KeyboardAnalyzerProvider.java:39]: Hey. If the analyzer was not created in the constructor, I received an error when loading the plugin. I can’t provide the error text. But the point was that the analyzer was created too early.

I looked at this code in the example of creating an analyzer from the elasticsearch repository.

Comment (aaerofeev) [reply src/main/java/org/elasticsearch/index/analysis/keyboard/KeyboardAnalyzerProvider.java:39]: Error similar like this
```
Analyzer is closed

**Review by npapakha — APPROVED** [review]

**Pull request merged**